src/Graph.tsx

```diff
@@ -457,7 +513,7 @@ class Graph extends React.Component {
                                 return 'octagon';
                             }
                             if (labels.indexOf('platform') > -1 || labels.indexOf('gillnet') > -1) {
-                                return 'square';
+                                return 'rectangle';
```

Comment:
This was a weird one - `'square'` clearly works, but was being marked as invalid from the types, and [isn't mentioned in their docs](https://js.cytoscape.org/#style/node-body) - `rectangle` yields the same result and is documented 🤷‍♂️